solr/solr-ref-guide/src/query-settings-in-solrconfig.adoc

```diff
@@ -172,24 +172,54 @@ This parameter sets the maximum number of documents to cache for any entry in th
 <queryResultMaxDocsCached>200</queryResultMaxDocsCached>
 ----
 
-=== useCircuitBreakers
+=== circuitBreaker
 
-Global control flag for enabling circuit breakers.
+This set of configurations control the behaviour of circuit breakers.
 
 [source,xml]
 ----
-<useCircuitBreakers>true</useCircuitBreakers>
+<circuitBreaker class="solr.CircuitBreakerManager">
+  <!-- All specific configs in this section -->
+</circuitBreaker>
 ----
 
-=== memoryCircuitBreakerThresholdPct
+To control whether Circuit Breakers are globally enabled, use the following flag:
+
+[source,xml]
+----
+<str name="enabled">true</str>
```

Comment:
standard syntax for enabled is 

`<circuitBreaker class="solr.CircuitBreakerManager" enabled="true">`